homeassistant/components/homeassistant_sky_connect/__init__.py

```diff
@@ -75,7 +75,12 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     if not usb.async_is_plugged_in(hass, matcher):
         # The USB dongle is not plugged in
-        raise ConfigEntryNotReady
+        hass.async_create_task(
```

Comment:
I think this need to be done as a task, instead of awaiting the result?

Reply:
Sounds safer, although I'm not sure if we have any dependencies like that currently.

Reply:
I checked, and we reload the integration in `async_set_disabled_by`. Then we need to use a task, since the integration must be setup first before it can be unloaded.